### ISSUE TITLE
`resin login` fails with an expired or invalid token

### ISSUE THREAD

**Issue by wrboyce** [resin-io/resin-cli#419]
I see this mostly when switching between resin environments.

```
% resin login
Looks like your session token is expired.

Please try logging in again with:

	$ resin login

If you need help, don't hesitate in contacting us at:

  GitHub: https://github.com/resin-io/resin-cli/issues/new
  Gitter: https://gitter.im/resin-io/chat

% resin login
______          _         _
| ___ \        (_)       (_)
| |_/ /___  ___ _ _ __    _  ___
|    // _ \/ __| | '_ \  | |/ _ \
| |\ \  __/\__ \ | | | |_| | (_) |
\_| \_\___||___/_|_| |_(_)_|\___/

Logging in to resinstaging.io
? How would you like to login? (Use arrow keys)
❯ Web authorization (recommended)
  Credentials
  Authentication token
  I don't have a Resin account!
```

**Comment by jviotti** [comment]
Yeah, the whole CLI authentication mechanism is a mess, because we still don't have proper API keys support. As a workaround we use session tokens which are interpreted as expired when we switch contexts.

I guess we can catch the expiration error and delete it and display the login dialog directly. Letting the user know that the token is expired and ask him to retry manually doesn't make much sense.

**Comment by pimterry** [comment]
Related to (arguably duplicates?) #443

**Comment by wrboyce** [comment]
@pimterry I think you're right, they look like duplicates.

**Comment by pimterry** [comment]
Fixed in 6.6.8
